### PR TITLE
Updated font file - Was missing characters

### PR DESCRIPTION
**Pull request by hassanshaikley**
: )

Should be perfect now!

### CODE REVIEW

File: client/fonts/5x5.bdf [hunk 1]
```diff
@@ -1050,44 +1049,56 @@ f8
...
+88
 00
 00
-00
-00
-f8
 ENDCHAR
-STARTCHAR \
-ENCODING 92
+STARTCHAR _
```
Comment (hzeller) [inline client/fonts/5x5.bdf]: You removed backslash ?

File: client/fonts/5x5.bdf [hunk 2]
```diff
@@ -396,11 +395,11 @@ SWIDTH 540 0
 DWIDTH 5 0
 BBX 5 5 0 -1
 BITMAP
-18
-30
+600
```
Comment (hzeller) [inline client/fonts/5x5.bdf]: What is up with these 600 and 300 ? Looks like accidental changes.

**Comment by hzeller** [comment]
mmh, the pipe symbol and curly braces are gone now ? {|}

**Comment by hassanshaikley** [comment]
Wait we had a pipe and curly? Let me take a look at that = )

**Comment by hzeller** [comment]
well, at least the diff indicates that. But maybe it was just mis-labelled ?

**Comment by hassanshaikley** [comment]
I didn't find them, but I implemented them now!

**Comment by hzeller** [comment]
Thanks, merged that manually now.
If you print out a text now, the [] characters still don't really show up. The [ looks like a ^, so that is probably mis-labelled. The ] does not show up at all.
(if you change that, make sure to first fetch/merge from upstream as the other change is already in)

**Pull request closed without merging**